CITATION.cff

```diff
@@ -0,0 +1,12 @@
+cff-version: 1.2.0
+title: 'CUDA Quantum'
+message: 'If you use this software, please cite it as below.'
+authors:
+  - name: "The CUDA Quantum development team"
+license: Apache-2.0
+license-url: "https://github.com/NVIDIA/cuda-quantum/blob/main/LICENSE"
+repository-code: "https://github.com/nvidia/cuquantum"
```

Comment:
@tlubowe Would you update this to cuda-quantum?

Reply:
updated - good catch